lib/perl/Genome/InstrumentData/Command/Import/Basic_cghub.t

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+BEGIN {
+    $ENV{UR_DBI_NO_COMMIT} = 1;
+    $ENV{UR_COMMAND_DUMP_STATUS_MESSAGES} = 1;
+};
+
+use above "Genome";
+
+require Genome::Utility::Test;
+require File::Compare;
+require File::Spec;
+use Test::More;
+
+use_ok('Genome::InstrumentData::Command::Import::Basic') or die;
+
+# Test data
+my $data_dir = Genome::Utility::Test->data_dir_ok('Genome::InstrumentData::Command::Import', File::Spec->catfile('cghub', 'v2')) or die;
+my $source_bam = File::Spec->catfile($data_dir, '387c3f70-46e9-4669-80e3-694d450f2919.bam');
+
+# Create library
+use_ok('Genome::Sample::Command::Import') or die;
+my $sample_importer = Genome::Sample::Command::Import::Tcga->execute(
+    name => 'TCGA-AB-2804-03B-01W-0728-08',
+);
+my $library = $sample_importer->_library;
+ok($library, 'created library') or die;
+
+# Local Bam
+my $cmd = Genome::InstrumentData::Command::Import::Basic->execute(
+    library => $library,
+    import_source_name  => 'CgHub',
+    source_files => [$source_bam],
+    instrument_data_properties => [qw/ target_region_set_name=none tcga_name=TCGA-AB-2804-03B-01W-0728-08 /],
+);
+ok($cmd->result, "execute");
+
+my @instdata =  $cmd->_new_instrument_data;
+is(@instdata, 1, 'create instrument data');
+my $instdata = $instdata[0];
+
+# Bam
+my $bam_path = $instdata->bam_path;
+ok(-s $bam_path, "bam exists");
+my $expected_bam_path = File::Spec->join($data_dir, 'expected.bam');
+is(File::Compare::compare($bam_path, $expected_bam_path), 0, 'bam matches');
```

Comment:
I don't know if comparing BAM files is already a common practice or not but comparing compressed files may not work since the files could differ bit-for-bit while still having the same content.  We may need to compare them in SAM format.  It looks like `File::Compare::compare` would support this by opening a handle to the BAM, piping through `samtools view`, and passing in the handles instead of the paths.  Again, I don't know if this is actually necessary.